samtranslator/model/api/http_api_generator.py

```diff
@@ -586,6 +585,27 @@ def _construct_stage(self):
 
         return stage
 
+    def _set_description(self):
```

Comment:
nit: when I read `set_xyz` I was expecting a setter which will get a value and sets the value of `xyz`. Can we change the method name to something else?

Reply:
@mndeveci how about `_add_description`? It only sets `description` when it's not present, so essentially it's "add" instead of "set". It also matches other methods within the class like `_add_cors(...)` and `_add_endpoint_configuration(...)`

Reply:
changed to `_add`